draft-marchan-kdl2.md

```diff
@@ -323,7 +363,7 @@ characters](#non-identifier-characters).
 
```

Comment:
The above should get section links too. I guess a number of these aren't even showing up in the GitHub diff